### ISSUE TITLE
Flow.Launcher.Plugin.ProcessKiller  ***Enhance***

### ISSUE THREAD

**Issue by watchingfun** [Flow-Launcher/Flow.Launcher#1317]
**Is your feature request related to a problem? Please describe.**

1. Plugin allows searching by tcp port
2. After the kill fails, try to execute it again with administrator privileges

**Additional context**

I tried the implementation myself, but I'm sorry, I'm not a c# programmer, the implementation might be ugly, but the features are really desperately needed
[https://github.com/Flow-Launcher/Flow.Launcher/commit/f984223f0234d6cc4d39bef7b4fbfd2dd3aef16b](https://github.com/Flow-Launcher/Flow.Launcher/commit/f984223f0234d6cc4d39bef7b4fbfd2dd3aef16b)

**Comment by nachmore** [comment]
When you say "search by port", do you mean for the process that has the port open?

**Comment by watchingfun** [comment]
> When you say "search by port", do you mean for the process that has the port open?

yes, Usually I want to kill a process, I may first only know its name and occupied port

**Comment by taooceros** [comment]
> [https://github.com/Flow-Launcher/Flow.Launcher/commit/f984223f0234d6cc4d39bef7b4fbfd2dd3aef16b](https://github.com/Flow-Launcher/Flow.Launcher/commit/f984223f0234d6cc4d39bef7b4fbfd2dd3aef16b)

The code is not that ugly. Feel free to fire a pr and we can adjust some of the style inside.
